File: 2st/Yeonwoo/마지막 두 원소.py
def solution(num_list):
    answer = []
    for i in num_list:
        answer.append(i)
    if answer[-1] > answer[-2]:
        answer.append(answer[-1]-answer[-2])
    elif answer[-1] <= answer[-2]:
        answer.append(answer[-1]*2)
    return answer

File: 마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int index = num_list.length - 1;
        int[] answer = new int[num_list.length + 1];

        for (int i = 0; i < num_list.length; i++) {
            answer[i] = num_list[i];
        }

        if (num_list[index] > num_list[index - 1]) {
            answer[index + 1] = num_list[index] - num_list[index - 1];
        } else {
            answer[index + 1] = num_list[index] * 2;
        }

        return answer;
    }
}
